Modules/Scripted/WebServer/WebServerLib/DICOMRequestHandler.py

```diff
@@ -61,12 +61,16 @@ def handleStudies(self, parsedURL, requestBody):
         splitPath = parsedURL.path.split(b'/')
         offset = 0
         limit = 100
+        patientID = None
         params = parsedURL.query.split(b"&")
         for param in params:
             if param.split(b"=")[0] == b"offset":
                 offset = int(param.split(b"=")[1])
             if param.split(b"=")[0] == b"limit":
                 limit = int(param.split(b"=")[1])
+            if param.split(b"=")[0] == b"PatientID":
```

Comment:
Let's accept 00100020, too:
```suggestion
            if param.split(b"=")[0] in [b"PatientID", b"00100020"]:
```